### PR TITLE
Add syft package!

### PR DESCRIPTION
**Pull request by dlorenc**
Signed-off-by: Dan Lorenc <dlorenc@chainguard.dev>

### CODE REVIEW

**Review by developer-guy — DISMISSED** [review]

File: syft.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  description: CLI tool and library for generating a Software Bill of Materials from container images and filesystems
+  target-architecture:
+    - all
+  copyright:
+    - license: Apache-2.0
+      paths:
+        - "*"
+  dependencies:
+    runtime:
+      - apk-tools
```
Comment (luhring) [inline syft.yaml:14]: What's the thinking behind this?

File: syft.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+environment:
+  contents:
+    packages:
+      - git
```
Comment (luhring) [inline syft.yaml:19]: Do we need `git`?

File: syft.yaml (hunk 1 above)
Comment (dlorenc) [reply syft.yaml:14]: Good question, I stole this from the apko sample of go/install, my mistake!

File: syft.yaml (hunk 2 above)
Comment (dlorenc) [reply syft.yaml:19]: Yea, go install fetches things with git.

Comment (dlorenc) [reply syft.yaml:16]: Well, I guess it doesn't always, but it can. I'll try without and see if it works here!

Comment (dlorenc) [reply syft.yaml:16]: nvm - it's installed anywhere here: https://github.com/chainguard-dev/melange/blob/main/pkg/build/pipelines/go/install.yaml#L8

**Review by luhring — APPROVED** [review]
Sweet! 🎉

**Review by rawlingsj — APPROVED** [review]

**Pull request merged**